io.github.complexlogic.EasyAudioSync.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.complexlogic.EasyAudioSync
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: easyaudiosync
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --filesystem=home
+  - --filesystem=/media
+  - --filesystem=/run/media
+  - --filesystem=xdg-run/gvfs
+modules:
+  - name: EasyAudioSync
+    buildsystem: cmake-ninja
+    config-opts:
+    - -DCMAKE_BUILD_TYPE=Release
+    - -DCMAKE_SYSTEM_NAME=Linux
+    - -DSTRIP_BINARY=ON
+    - -DQT_VERSION=5
+    - -DVCPKG=ON
+    sources:
+      - sources.yml
+    build-options:
+        build-args:
+        - --share=network
```

Comment:
We don't allow network access during build.

Reply:
The reason I've asked for network access is because my build script relies on the CMake `FetchContent` module which requires an external connection.

Are there any exceptions allowed to this rule?

Reply:
no. You need to download them by declaring them in the manifest.

Reply:
OK. In that case, I have to close this and figure out an alternative solution.